### DISCUSSION TITLE
paged.js options

### DISCUSSION THREAD

**Discussion by junedtan** [thegetty/quire#806] [Development]
I just tried to build pdf and it turns out to be suuuper large, caused by so many images inside it. The images themselves are unfortunately fetched directly from external resource (like, the "src" property on figures.yaml points to an external URL) and there is no option in the URLs to like fecthing a certain quality.

I found out these wkhtmltopdf options that might bring the file size down: https://stackoverflow.com/questions/33528780/any-way-to-reduce-file-size-using-wkhtmltopdf. I wonder if Paged.js has the similar options (can't find them here: https://pagedjs.org/documentation/2-getting-started-with-paged.js/#command-line-version) and how to set those option in Quire (maybe quire pdf --something or an entry in config.yaml?). If there's none, at least where (which file among Quire source code) can I find the paged.js command being called so maybe I can add some options there?

**Comment by Erin-Cecele** [comment]
@junedtan Thank you for your inquiry. @fchasen do you have any advice?

**Comment by Erin-Cecele** [comment]
Hi @junedtan, the solution in this instance is to update to the latest version of 11ty, which is included with all projects created using the command `quire new`. You can follow these instructions in our documentation: https://quire.getty.edu/docs-v1/install-uninstall/#update-an-existing-project. Let us know if the issue persists after updating. Thank you!
